packages/sample-app-fabric/tsconfig.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "@react-native/typescript-config/tsconfig.json"
```

This should match other apps in our mono repo.
```suggestion
  "extends": "@rnw-scripts/ts-config",
  "exclude": [
    "node_modules"
  ]
```